core/tracing/tracer.go

```diff
@@ -0,0 +1,78 @@
+package tracing
+
+import (
+	"time"
+
+	"go.uber.org/fx/core/ulog"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/uber-go/tally"
+	jaegerconfig "github.com/uber/jaeger-client-go/config"
```

Comment:
why alias this? generally we should only alias a package if it conflicts with another import, but we're not importing another `config` package here, right?